### PR TITLE
Add virtual channel tutorial

### PR DESCRIPTION
**Pull request by matthiasgeihs**

### CODE REVIEW

File: source/go-perun/virtual_channel_tutorial/index.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Source code
+===========
+
+As an example we will rely on the `virtual channel test <https://github.com/hyperledger-labs/go-perun/blob/v0.9.1/client/virtual_channel_test.go>`_ of Go-Perun. This test shows how a virtual channel is created and used.
+
+Channel opening
++++++++++++++++
+
+As a pre-requisite we require that Alice and Bob have an open channel with Ingrid. These channels are created in the beginning of the test setup.
```
Comment (cryptphil) [inline source/go-perun/virtual_channel_tutorial/index.rst:32]:
```suggestion
As a pre-requisite, we require that Alice and Bob have an open channel with Ingrid. These channels are created at the beginning of the test setup.
```

File: source/go-perun/virtual_channel_tutorial/index.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    :language: go
+    :lines: 246-261
+
+Virtual channel proposals are created using ``client.NewVirtualChannelProposal`` and then specifying the identifiers of the channels from which the funds come from.
+
+.. literalinclude:: ../source/go-perun/client/virtual_channel_test.go
+    :language: go
+    :lines: 258
+
+Furthermore, it must be specified how the indicies of the participants map from the parent channels to the virtual channel. Here we specify that Alice has index in the virtual channel and Bob has index 1.
```
Comment (cryptphil) [inline source/go-perun/virtual_channel_tutorial/index.rst:46]:
```suggestion
Furthermore, it must be specified how the indices of the participants map from the parent channels to the virtual channel. Here, we specify that Alice has index in the virtual channel and Bob has index 1.
```

File: source/go-perun/virtual_channel_tutorial/index.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    :language: go
+    :lines: 251-252
+
+The channel proposal is sent as usual by using ``client.ProposeChannel``.
+
+.. literalinclude:: ../source/go-perun/client/virtual_channel_test.go
+    :language: go
+    :lines: 263
+
+Afterwards a channel object is returned that can be used for off-chain transactions just as any other channel object.
```
Comment (cryptphil) [inline source/go-perun/virtual_channel_tutorial/index.rst:59]:
```suggestion
Afterwards, a channel object is returned that can be used for off-chain transactions just as any other channel object.
```

**Review by cryptphil — APPROVED** [review]
Just added some comments regarding punctuation, feel free to apply them if you like.

**Comment by matthiasgeihs** [comment]
Thanks for the review, applied your suggestions @cryptphil

**Review by cryptphil — APPROVED** [review]

**Pull request merged**